SpectralCubeReprojectExample.ipynb

```diff
@@ -0,0 +1,995 @@
fmO8m9Uym4scImLtbmOcecwA1rgnZYtYgfiPfexjpcTJZoLIjXZ9fR1XV1elROCUx+817ZgHEOUA2tHDjohHMoELms/YeFo+wZjgZBr61JsbA2ZSd5hRB7zDPtIunpNoU/p2OByWcuC5XwzYAOz5BoaxYnDFO8L81Ov1Ut/u7e3FyclJdLvddA/Kcnt7G5eXl1Gr1WJnZyd5IzDmK1hPe2oajdU52pSFed5zmDdEfGa3a2Xff6vA3AfUvvjFL8af/dmfxf7+fnoBSS3yt3/7t4l940W2hiJPxMnfWThM05OUmMmf70dE0qA5xxi7/4jycV+NRiMtGBcXF9HpdFISYtxVEeuoVJ7ve8MY4grKmR1cmNagRKwnd55Xr9fv6ALNRHlR8C47X7hoC0/yEeX0CdgmN4+TA9Oejs4kAo7ncH9y8Bkk8RtNjp9H/afTaYpm5vuO2sTVl5eTyEbruPJozNzt6XQe7geDGC+CuS7PmjYAqRlaymfgkPePtVE5cBwOh6WkwXzf19m17s1KHoXqhdjRgQZDbIJgd7yYstnZ1KfWflkT6T5ijDoyEzCOO5VEwBFx5/kAQeqAC5/xDwBmE0XdcyCeu2adEgMw53yCvg+suetLPWALYb4oMyymZSIRkdg1jvgaj8epLxnL9IfBHLpK6wKpe71eL0XK5xHAsKq8t35eo9GInZ2dFGVLTri9vb2SphaQxf/JIrC/v5/mN/qPPmQj6bkGDXS3242iKJJGDyO11WAwSM9jbvzMZz4TL6K9X8xcrVb7iYj43YhoRMS/LIri89nntXc+/68iYhIR/11RFF+t1WqvRcQfRMSDiFhGxO8XRfGeyfsqMPc9slqt9q8i4p/9oMvxDzF0Xna3ofM4Pz+P/f39lDeIyfv6+joJdlmELFzONSaIjC0A986c3Rz3c64u3DUwPOyOPWHzPe7farVKQRocFRZRZtYsZI5YuznZQQNmqZP1bLAOfj4sYH5P67YMaFigaFcDE4MNrrGGjOc6CCQiSnopdDcGDA44cfAAgMVsBYuej53iyCPaol6vR6fTSQuvXX88x4uIAR6sZJ4qxC5pAJ3dpXa5ug7+e65ZBNzyTLuDuD5nSvM65M9jI8A4MgtD7jLfxykxcgbJDA0bHxggAy+7TA12GE+cKGGXL4t8zsTyXMAdIJPx1e124+bmJi4vL++877C3Lid1yE/W8Lim/rBrZsYNiilvDuYYm653rsm09pS2ouweZ2yguK8DZgCAg8EgBanYXeqNlEG+2a482ASwbdBIOU9PT+P09DQ6nU5iO32kHG3S6XTi6OgoeUsAWcgirFnmXUGTaf1zp9NJm5379+9Hp9NJc/Kbb74Zo9Eojo+PE2jblHB7a2srpW6JWJ35TWqXF8Fqtdo4Iv64KIr37ZiKWq3WiIj/NSJ+PCLeioh/V6vV/m1RFH+tr/2XEfHxd37+i4j4F+/8nkfEp94Bdp2I+H9rtdr/lV1bsooT/R5ZURSfLIpi/zt/s7LKKqusssoq+0FZURT7ADkA+D/057uw/zwi/kNRFH9fFMVNRPzriPin2Xf+aUT8QbGy/zsiDmq12oeKonhcFMVX3ynfMCL+JiJefa+HVczcB9j+4i/+In70R380uTat7WG35iS+uBDG43FJ64NbwYdx5+kBer1eYlp4Bi4ZmEEHJMzn88QCweDku3BcqrgCeR4uM++0qR8uJVyK3v2bdXFEINFndgPZFUr0p/VOEZGO5qE9eC51oB5khM+ZGjMJm/RR8/k8RfnRR+z+LW7nu7A2uKXNXKF1tJuT525ipOg/2o0+ypOy4vI2Y0bdYDByzVzOouTtYp0jn+UpXzxezEThssqF2k61kbNMuAztOoVdow+tH0ITlbvEGRPcj7+7fPQT7e7ksXkQhxlImFhcZs7Ev7e3V2KWHRQDe8yPGWee1e/3S/VptVqxWCxiMBjEeDxO7wfmiEbc7LQL9c6lB/S7x4JZO2v7IspuZOsZPYfxPti96utgCHOGjZQslmtYikB5YG6t4SSwBzem+4hoZbS51P3w8DCOj4/j8PAwtre3S2esMl4Gg0G6HoaNdC4R67NmHU3e6/VSHzIvMB7a7Xa8+uqrce/evWi1WklOExHxjW98I7UjqWmw2WyW5vHb29s4PT2NiBW7SHTui2C1Wu2fR8Q/jyhLe/6BdlKr1b6i//9+URS/r/+/GhFv6v9vxYp1i+/wnVcj4rHK+tGI+E8i4v95r8JUYO4DbG+88UZ8/vOfTwEQRIbxslsQjcvOQAo9yGQySS99u90uLR4Aq0ajkc5x5LPRaJSo/DxVBxOjXYjWJnnxQBMSEQmAOoWB3bostOhNnNYDl+B8Pk/6Mwyt3aYdGeDPLhHqTpvm4noWahZTu/fye/szA9AcmNDO1jDmoMxg0nVBj4b+yiCChR6w4UhXC7rtBmWMvBtAtGvSbjG7gK2ZxOijHHwY7Pq378vfDYgtRHcb0r4el3abu2/yPqL8fLcoihKAspsxDxpBWkC70DbkAjPA4B0giMjPo4/IkUfakvy0CsoASGfMk5cM3RyLd0QkINPv9+PZs2dxeXl5R2ZBezFmMLu3rTej3QzAHJVKuXn3/a45GMKBOIvFolTnfEwgD6EN6D+0woB7z1muH6DcdecYNM8/lM0RqwZXTvkDUDYIzF3HnrNIP3J1dRU3NzcpspWADuqFxCIi4mMf+1h8+MMfjg996ENxdHQUjUYjpY26vb2NVqsV/X4/rq6uSvMaVryjd7W8ZjKZxCc+8Yl4Uewd0PX7ERHHx8fFP1Iz96woiv/0PT7fdBRG/qD3/E6tVmtHxB9FxC8URTF4r8JUYO4DbvP5PN58c7UxaDRWSYNPTk7SjtCRqOi38nB9gJ5F6wZzLIbz+epoGHZCDx48KP3du0Mi1sixZEbISUKtI4pYTWKwAZQpZ98AfHm0XcR6553nRGu32+n77E55LgALIJgnMfXh3wY7tBegLxems2A44CKvk9s6IkoL
```

Comment:
```suggestion
    "Note that it is important for the to-be-interpolated cube, in this case `cube2`, to have pixels bounding `cube1`'s spectral axis, but in this case, it does not.  If the pixel range doesn't overlap perfectly, it may blank out one of the edge pixels.  So, to fix this, we add a one-channel buffer:"
```

Reply:
Not sure why this switched lines on me... should be 1 line up.